Mathlib/Data/HashMap.lean

```diff
@@ -29,14 +29,17 @@ namespace Batteries.HashMap
 variable [BEq α] [Hashable α]
 
 /-- The list of keys in a `HashMap`. -/
+@[deprecated "Cleaning up unused API additions in Mathlib." (since := "2024-06-12")]
```

Comment:
What do you think about: "This addition to mathlib is unused: if you need this, please file at issue in the `batteries` repository."?

Reply:
Just noting that `Batteries.HashMap` is planned for wholesale deprecation shortly, but sure.

Reply:
Is there a tracking issue / Zulip discussion for that?

Reply:
Not that I'm aware of.